src/style.css

```diff
@@ -1 +1,15 @@
-/* styles go here */
+.labeles {
```

Comment:
The class name 'labeles' seems to be a typo. Consider changing it to 'label' for better readability 📝